app/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,7 @@
     >
 
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.INTERNET"/>
 
     <application
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/ar-localizer-view-android/blob/6e7a43f049fda9487b1419ac862329af39e581fa/app/src/main/AndroidManifest.xml#L9"></span>App is not indexable by Google Search; consider adding at least one Activity with an ACTION-VIEW intent filter. See issue explanation for more details.</td>
    </tr>
  </tbody>
</table>